site/content/kapp-controller/docs/develop/app-spec.md

```diff
@@ -209,6 +209,23 @@ spec:
           - configMapRef:
               name: cfgmap-name
           - path: values/shared.yml
+          # downwardAPI allows passing info about App CR as values into templates (v0.39.0+)
+          - downwardAPI:
+              items:
+              - name: namespace
```

Comment:
i think it would be great to add a comment to name and fieldPath fields indicating format and any nuances (e.g. dots in the name, escpaing dot works in the name, fieldPath is a "relaxed" jsonPath.) i would also recommend just having a one or two examples (instead of 7) to shorten the page.

Reply:
good ideas. done